MagickCore/CMakeLists.txt

```diff
@@ -688,8 +689,10 @@ if(WEBP_DELEGATE)
 endif()
 if(WINGDI32_DELEGATE)
   list(APPEND MAGICKCORE_CODERS_DELEGATES_SRCS ${CODERS_PATH}/clipboard.c ${CODERS_PATH}/emf.c)
-  # Seems that emf.c uses namespaces so we need to compile as c++
-  set_source_files_properties(${CODERS_PATH}/emf.c PROPERTIES LANGUAGE CXX)
+  if(BUILD_MAGICKPP)
```

Comment:
Should this if also contain the above line? And the same for the openexr delegate?